designer/client/component-create.js

```diff
@@ -6,6 +6,16 @@ import ComponentTypes from 'digital-form-builder-engine/src/component-types'
 class ComponentCreate extends React.Component {
   state = {}
 
+  componentDidMount () {
+    const { data, onInit } = this.props
+    data.getId().then(id => {
+      this.setState({ id: id })
```

Comment:
just `this.setState({ id })` should work

Reply:
Fixed